internal/integration/helpers.go

```diff
@@ -37,10 +42,27 @@ import (
 	exportmodel "github.com/google/exposure-notifications-server/internal/export/model"
 )
 
-const (
-	exportDir = "my-bucket"
+var (
+	exportDir    = "my-bucket"
+	fileNameRoot = "/"
 )
 
+func GetExportDir() string {
```

Comment:
I don't think you need to make getters or setters for this. Once it's a variable, it's globally accessible anyway to all the tests.

Reply:
sg, done